### PR TITLE
Add storybook for TopInstructions

### PR DESCRIPTION
**Pull request by dmcavoy**
Creates a storybook for TopInstructions. There are still some features I would like to see added in the future but this covers the major areas.

Also move the audio button to the left side when in RTL for CSD/CSP since I saw that issue while using storybook.

Screen Shot 2019-06-12 at 11 38 44 AM

### CODE REVIEW

File: apps/src/templates/instructions/TopInstructions.jsx [hunk 1]
```diff
@@ -133,6 +133,22 @@ const audioStyle = {
...
+    marginTop: '3px',
+    marginBottom: '3px'
+  },
+  buttonImg: {
+    lineHeight: '24px',
+    fontSize: 15,
+    paddingLeft: 12
+  }
+};
+
```
Comment (Erin007) [inline apps/src/templates/instructions/TopInstructions.jsx:151]: yeah! Thanks for including these styles. RTL doesn't get enough love on our site

File: apps/src/templates/instructions/TopInstructions.story.jsx [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+  'CSD/CSP Instructions Right-to-Left': {
+    rtl: true,
+    isCSF: false
+  }
+};
+
+export default storybook => {
+  const stories = storybook.storiesOf('TopInstructions', module);
+
+  Object.entries(STORIES).forEach(([name, options]) => {
```
Comment (Erin007) [inline apps/src/templates/instructions/TopInstructions.story.jsx:144]: oh I like this set up - I don't think I've seen it before, but seems tidier than writing the stories out individually and passing different props combos

**Review by Erin007 — APPROVED** [review]

File: apps/src/templates/instructions/TopInstructions.story.jsx (hunk 2 above)
Comment (dmcavoy) [reply apps/src/templates/instructions/TopInstructions.story.jsx:144]: Can't take credit for it - copied the setup that existed for InstructionsCSF

**Review by maddiedierker — APPROVED** [review]
🥇great work! this is a really useful component to have available in storybook

**Pull request merged**